src/app/pages/learning-research/components/members.js

```diff
@@ -0,0 +1,252 @@
+                    </p>
+                </div>
+
+                <div label='Collaborating Researchers'>
+                    <CollaboratorGrid>
+                        {collaboratingResearchers.map((member, index) =>
+                            <Member
+                                member={member}
+                                key={index}
+                                displayName={`${member.firstName} ${member.lastName}`}
+                            />
+                        )}
+                    </CollaboratorGrid>
+                </div>
+
+                <div label='Alumni'>
+                    <AlumniGrid>
+                        {alumni.map((alumnus, index) =>
+                            <Alumnus alumnus={alumnus} key={index} />
+                        )}
+                    </AlumniGrid>
+                </div>
+            </TabAccordionCombo>
+        </Section>
+    );
+};
+
+const MemberModal = ({ member, isOpen, onHide }) => {
+    return (
+        <Dialog className='member-modal' onPutAway={onHide} isOpen={isOpen} closeOnOutsideClick={true}>
+            <MemberDetails member={member} />
+        </Dialog>
+    );
+};
+
+export const Member = ({ member, displayName }) => {
+    const [show, setShow] = useState(false);
+
+    return (
+        <Box direction='column' align='center' className='text-center'>
+            <img
+                height={145}
+                width={145}
+                alt={member.photo.title}
+                src={member.photo.file}
+                tabIndex={0}
+                onClick={() => setShow(true)}
+            />
+            <span
+                css={{marginTop: '1.5rem', textAlign: 'center'}}
+                className='link-text'
+                tabIndex={0}
+                onClick={() => setShow(true)}
+            >
+                {displayName}
+            </span>
+            <small>{member.title}</small>
+            <MemberModal member={member} isOpen={show} onHide={() => setShow(false)} />
+        </Box>
+    );
+};
+
+export const MemberDetails = ({ member }) => {
+    return (
+        <Box
+            css={{
+                fontSize: 15,
+                margin: '20px'
+            }}>
+            <Box gap='large'>
+                <MemberImage className='desktop-only' src={member.photo.file} alt={member.firstName} />
+                <Box direction='column' gap='large'>
+                    <MemberInfo member={member} />
+                    <MemberEducation member={member} />
+                    <MemberResearchInterest member={member} />
+                    <MemberBio member={member} />
+                    <MemberLinks member={member} />
+                </Box>
+            </Box>
+        </Box>
+    );
+};
+
+const MemberImage = styled.img({
+    marginBottom: '.5rem',
+    width: 145,
+    height: 145,
+    [media.mobile]: {
+        width: 75,
+        height: 75,
+        marginRight: '2rem'
```

Comment:
instead of using marginRight to space, should it use flexbox `gap` property?  That also has the advantage of spacing vertically if the images wrap